assembl/static2/js/app/components/debate/common/editPostForm.jsx

```diff
@@ -43,7 +43,8 @@ type EditPostFormProps = {
   bodyMaxLength?: number,
   draftable?: boolean,
   draftSuccessMsgId?: string,
-  isDebateModerated: boolean
+  isDebateModerated: boolean,
+  multiColumns: boolean
```

Comment:
The EditPostForm is used as several places. It should be `multiColumns?: boolean` and add a default prop `multiColumns: false`

Reply:
Sure